dataland-frontend/src/components/forms/parts/elements/basic/MultiSelectFormElementBindData.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <MultiSelect
+    v-model="selections"
+    :options="options"
+    :placeholder="placeholder"
+    :show-toggle-all="false"
+    :class="innerClass"
+    :optionValue="optionValue"
+    :optionLabel="optionLabel"
```

Comment:
PrimeVue says that this property is of type `string |Function`.
Therefore we should stick to those types.   Currently we use the type `type: [String, Boolean] as PropType<string | boolean>` on it.

PrimeVue also says that if you want the `optionValue` defaults to the option itself if we don't pass anything for it here. So why not just ignoring it for now?   
PrimeVue says:
"Note that, when options are simple primitive values such as a string array, no optionLabel and optionValue would be necessary. "

Reply:
yes, righ.

Reply:
done

Reply:
![image](https://github.com/d-fine/Dataland/assets/99255296/b02ca677-aaf8-4c30-8597-bdcddc81cd86)

The red marked prop `optionLabel="label"` is the default, and therefore it is not needed to explicitly set it.
Please remove it.

Reply:
![image](https://github.com/d-fine/Dataland/assets/99255296/6b8ddfe3-b612-435c-a026-962eba39a96c)

The `PropType` types should be `string` and `Function` instead of `string` and `boolean`

Reply:
Since setting the type as Function causes an EsLint error, I left the type unassigned PropsType

![image](https://github.com/d-fine/Dataland/assets/96523011/c607ed7c-c56c-4c1c-9c94-0cab95586331)